src/main/kotlin/org/jetbrains/kotlinx/jupyter/htmlUtil.kt

```diff
@@ -0,0 +1,76 @@
+package org.jetbrains.kotlinx.jupyter
+
+import org.jetbrains.kotlinx.jupyter.api.VariableState
+
+// TODO : perhaps, create factory class
+fun generateHTMLVarsReport(variablesMap: Map<String, VariableState>): String {
```

Comment:
Same thing about naming. Please find all the usages of `variablesMap` and `varsMap` and think about better names